### PR TITLE
feat(CX-2888): add submission status to the MyC Artwork page

### PR DESCRIPTION
**Pull request by dariakoko**
The type of this PR is: **Feat**

This PR solves [CX-2888]

### Description

https://user-images.githubusercontent.com/36167539/213417088-d7af33f5-0cf2-4dfe-a84c-11ab73942ad8.mov

[GRO-434]: https://artsyproduct.atlassian.net/browse/GRO-434?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

[CX-2888]: https://artsyproduct.atlassian.net/browse/CX-2888?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by lordkiz — APPROVED** [review]
LGTM

**Review by MrSltun — APPROVED** [review]
Looks great! 🌟

File: src/Apps/MyCollection/Routes/MyCollectionArtwork/Components/MyCollectionArtworkSWASectionSubmitted.tsx [hunk 1]
```diff
@@ -1,13 +1,109 @@
...
+import { Media } from "Utils/Responsive"
+
+// TODO:- We are using displayText for Statuses for now. Consider changing the logic when proper statuses are made available on Metaphysics.
+// See https://artsyproduct.atlassian.net/browse/SWA-217
+// same as on Eigen
+const STATUSES: { [key: string]: { color: string; text: string } } = {
+  "submission in progress": { color: "yellow150", text: "In Progress" },
+  "submission evaluated": { color: "orange150", text: "Evaluation Complete" },
+  sold: { color: "black100", text: "Artwork Sold" },
+}
```
Comment (MounirDhahri) [inline src/Apps/MyCollection/Routes/MyCollectionArtwork/Components/MyCollectionArtworkSWASectionSubmitted.tsx:23]: **suggestion:** These statuses should better be coming from Metaphysics IMO (we can store the color as a hex code as well). I know this logic exists as well in Eigen so I would recommend addressing this in a different ticket.

File: src/Apps/MyCollection/Routes/MyCollectionArtwork/Components/MyCollectionArtworkSWASectionSubmitted.tsx [hunk 2]
```diff
@@ -1,13 +1,109 @@
...
+
+  const isMyCollectionPhase8Enabled = useFeatureFlag(
+    "my-collection-web-phase-8-submission-status"
+  )
 
-export const MyCollectionArtworkSWASectionSubmitted = () => {
   const article =
     "https://support.artsy.net/hc/en-us/sections/360008311913-Sell-with-Artsy"
 
+  const approvedDisplayText = STATUSES[submissionStatus!.toLowerCase()]?.text
```
Comment (MounirDhahri) [inline src/Apps/MyCollection/Routes/MyCollectionArtwork/Components/MyCollectionArtworkSWASectionSubmitted.tsx:44]: **thought:** is this text on for approved submissions? if not I would suggest maybe renaming this 👀

Comment (MounirDhahri) [reply src/Apps/MyCollection/Routes/MyCollectionArtwork/Components/MyCollectionArtworkSWASectionSubmitted.tsx:44]: maybe `displayText` or so

File: src/Apps/MyCollection/Routes/MyCollectionArtwork/Components/MyCollectionArtworkSWASectionSubmitted.tsx [hunk 3]
```diff
@@ -27,3 +123,44 @@ export const MyCollectionArtworkSWASectionSubmitted = () => {
...
+      <Text as="li" variant="sm-display" mt={2}>
+        Evaluation Complete - our specialists have reviewed this submission and
+        determined that we do not currently have a market for it.
+      </Text>
+      <Text variant="sm-display" mt={2}>
+        Have a question?
+      </Text>
+      <Text variant="sm-display">
+        Visit our help center or get in touch with one of our specialists at
+        sell@artsymail.com.
```
Comment (MounirDhahri) [inline src/Apps/MyCollection/Routes/MyCollectionArtwork/Components/MyCollectionArtworkSWASectionSubmitted.tsx:162]: **suggestion:** maybe we can make this email link clickable

**Review by MounirDhahri — DISMISSED** [review]
LGTM, i left few small comments but none are blocking

File: src/Apps/MyCollection/Routes/MyCollectionArtwork/Components/MyCollectionArtworkSWASectionSubmitted.tsx (hunk 3 above)
Comment (dariakoko) [reply src/Apps/MyCollection/Routes/MyCollectionArtwork/Components/MyCollectionArtworkSWASectionSubmitted.tsx:162]: And it should be sell@artsy.net 👍

File: src/Apps/MyCollection/Routes/MyCollectionArtwork/Components/MyCollectionArtworkSWASectionSubmitted.tsx (hunk 1 above)
Comment (dariakoko) [reply src/Apps/MyCollection/Routes/MyCollectionArtwork/Components/MyCollectionArtworkSWASectionSubmitted.tsx:23]: Yes, it thought of addressing it in a different ticket

File: src/Apps/MyCollection/Routes/MyCollectionArtwork/Components/MyCollectionArtworkSWASectionSubmitted.tsx (hunk 3 above)
Comment (MounirDhahri) [reply src/Apps/MyCollection/Routes/MyCollectionArtwork/Components/MyCollectionArtworkSWASectionSubmitted.tsx]: Thanks Daria

**Review by MounirDhahri — APPROVED** [review]
Thanks!

**Comment by MounirDhahri** [comment]
Please feel free to merge once you fix the build

**Pull request merged**